libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -1068,6 +1068,8 @@ - (void)setCurrentUser:(SFUserAccount*)user {
                 [self willChangeValueForKey:@"currentUser"];
                 _currentUser = user;
                 [self setCurrentUserIdentity:user.accountIdentity];
+                if (user.credentials.domain)
```

Comment:
I hope this isn't solving a problem, and is more of a nice to have.  `SFUserAccountManager.loginHost` should not be driving actions for any existing user, and if it is, such a dependency should be rectified/eliminated.

Reply:
I should add that the other change in this PR looks like it may be rectifying exactly such a case?  I don't see any other obvious areas where `loginHost` would be used against this suggestion, so that's encouraging.

Reply:
Is this setting the loginHost for the userAccountManager to the current users domain?

Reply:
Yes.  As part of setting `currentUser`—which *should* entail a guarantee of a valid `SFUserAccount` *and* its underlying credentials—this code would change `loginHost` to reflect the current user's login host.